### PR TITLE
Add coverage reports

### PR DESCRIPTION
**Pull request by na9da**
Adds coverage reporting to `mobx` branch.

### CODE REVIEW

File: buildprocess/configureWebpack.js [hunk 1]
```diff
@@ -107,6 +107,15 @@ function configureWebpack(terriaJSBasePath, config, devMode, hot, MiniCssExtract
         ]
     });
 
+    config.module.rules.push({
+      test: /\.(ts|js)x?$/,
+      include: [path.resolve(terriaJSBasePath, "lib")],
+      use: {
+        loader: "istanbul-instrumenter-loader"
```
Comment (kring) [inline buildprocess/configureWebpack.js]: `configureWebpack` is used to build TerriaMap, too, so won't this add instrumentation to apps, too? Or is that not a problem?

Comment (kring) [reply buildprocess/configureWebpack.js]: Actually at the moment it's breaking the app build completely. See the Deployment log.

Comment (na9da) [reply buildprocess/configureWebpack.js]: I have made the configuration package local. Currently, it only reports classes referenced via the tests!

**Comment by kring** [comment]
Looks good and I'm merging it. Just wondering, though: can/should we add the istanbul reporter to karma-saucelabs.conf.js too so that we get coverage results from Travis?

**Pull request merged**